services/user/Events/src/REvents.cpp

```diff
@@ -1063,8 +1063,9 @@ void load_tables(sqlite3* db, std::string_view sql)
    }
 }
 
-std::vector<char> REvents::sqlQuery(const std::vector<char>& query)
+std::vector<char> REvents::sqlQuery(const std::string& squery)
 {
+   std::vector<char> query = {squery.begin(), squery.end()};
```

Comment:
Converting to a string and then back to a vector is kind of odd. The implementation can use `span<const char>` or `string_view`--no copies needed.